src/Serialization/main.jl

```diff
@@ -864,3 +843,35 @@ function load(filename::String; params::Any = nothing,
     return load(file; params=params, type=type, serializer=serializer)
   end
 end
+
+export load_attrs,
+  load_object,
+  save_attrs,
+  save_object,
+  type_params,
+  load,
+  save,
+  @register_serialization_type,
+  DeserializerState,
+  SerializerState,
+  TypeParams,
+  encode_type,
+  haskey,
+  load_array_node,
+  load_node,
+  load_ref,
+  save_as_ref,
+  save_data_array,
+  save_data_basic,
+  save_data_dict,
+  save_data_json,
+  serialize_with_id,
+  set_key,
+  with_attrs
+
+end # module Serialization
+
+using Oscar.Serialization
+import Oscar.Serialization: load_object, save_object, type_params
+import Oscar.Serialization: load, save
```

Comment:
```suggestion
```
why do we need to import these? If I see correctly, they are already available due to the submodule exporting it.

Reply:
Where outside of the Serialization submodule do we add new methods to them? In particular load and save don't seem that they should be extended

Reply:
`experimental/FTheoryTools`, `experimental/QuadFormAndIsom`

Reply:
I agree for the first line but not for the second